src/Picqer/Financials/Exact/Query/Findable.php

```diff
@@ -5,8 +5,14 @@
 
     public function find($id)
     {
+        if ($this->primaryKey === 'Code') {
```

Comment:
Thank you.

It's a little nicer to change it to not use an if statement. This way it has a clear default.

```php
$filter = $this->primaryKey . " eq guid'$id'";

if ($this->primaryKey === 'Code') {
    $filter = $this->primaryKey . " eq $id";
}
```